adapters/adocean/adocean.go

```diff
@@ -196,7 +196,7 @@ func (a *AdOceanAdapter) makeURL(
 	slaveSizes map[string]string,
 	consentString string,
 ) (*url.URL, error) {
-	endpointParams := macros.EndpointTemplateParams{Host: params.EmitterDomain}
+	endpointParams := macros.EndpointTemplateParams{Host: params.EmitterPrefix}
 	host, err := macros.ResolveMacros(a.endpointTemplate, endpointParams)
```

Comment:
As mentioned  in https://github.com/prebid/prebid-server/pull/2910/files#r1257850255, this might be a breaking change.

Consider scenario where incoming request has `EmitterDomain` and not `EmitterPrefix`. Since `EmitterPrefix` is required param, so in this case prebid  server, will reject entire auction request. 

May be should consider some transition time for this change for backward compatibility.

Reply:
Yes that's true, but as I mentioned before we are in contact with our clients and decided to skip transition time entirely.

Reply:
Skipping the transition entirely is only possible if they remove you from their configs first, then the PBS host updates, and after that they add you back in. If an adapter's config fails the schema check, the whole request fails (all adapters, so no prebid happens at all). A transition period gives the publishers a window to update their configs. You can respond with an error and not call your bidder if the request is missing the new version of the parameter, we just want the published schema to allow the old version of the bidder config. It is only this schema check that triggers a complete failure of the request.

Reply:
OK, I haven't known that, I've updated pull request, is it OK now?

Reply:
Yes, thank you. After a transition period you can tighten down the schema again.